gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/client/graph/ValidateGraphHostOperation.java

```diff
@@ -49,19 +52,23 @@ public void execute() throws GraphOperationException {
             validateSystemIsUp(response);
 
         } catch (final WebClientRequestException e) {
+            LOGGER.error("Get Status request for {} ", proxySubGraph.getGraphId() + " failed. Reason: {}. ", e.getMostSpecificCause().getMessage() + " at {}", e.getUri(), e);
```

Comment:
I don't think this will work properly, see [SLF4J's parameterized logging](https://www.tutorialspoint.com/slf4j/slf4j_parameterized_logging.htm). It should look more like:
```java
LOGGER.error(String.format("Get Status request for %s failed. Reason: %s at %s", proxySubGraph.getGraphId(), e.getMostSpecificCause().getMessage(), e.getUri()), e);
```

Reply:
The same applied below, see [here](https://www.slf4j.org/api/org/slf4j/Logger.html#warn-java.lang.String-java.lang.Object...-) for using parameterized logging and [here](https://www.slf4j.org/api/org/slf4j/Logger.html#warn-java.lang.String-java.lang.Throwable-) for sending the exception.

Reply:
@t92549  Made the changes as per review comments

Reply:
Apologies to go back on what I said but you can actually do the exceptions still with paramterized logging, so something like this:
```java
LOGGER.error("Get Status request for {} failed. Reason: {} at {}", proxySubGraph.getGraphId(), e.getMostSpecificCause().getMessage(), e.getUri(), e);
```

Reply:
Done